### PR TITLE
Add post-review-requirement paragraph to the README

### PR DESCRIPTION
**Pull request by kcharter**
Now that I've configured the protected master branch to require an approving review
on a PR, I can no longer push directly to `master`. Good!

OK, here's a PR for the change I intended to push. Can I self-review?

### CODE REVIEW

**Review by jlish-granify — APPROVED** [review]
This code is in pristine condition

**Review by dfriel — APPROVED** [review]

**Review by jlish-granify — CHANGES_REQUESTED** [review]
Oh wait I change my mind

File: README.md [hunk 1]
```diff
@@ -7,3 +7,7 @@ Here's another paragraph, added after I made `master` protected. The
 branch requires pull request reviews before merging, except for
 administrators. Can I push directly?
 
+Hmm. Yes, I can push directly. Is it because I'm an administrator?
+OK, I've changed the configuration so we require pull request reviews
+for administrators too. Can I still push directly?
+
```
Comment (dfriel) [inline README.md:13]: Is this extra blank line necessary?

**Review by jlish-granify — APPROVED** [review]
Code looks good

File: README.md (hunk 1 above)
Comment (kcharter) [reply README.md:13]: Hey, this is definitely _the_ place to bikeshed!

Now this gives me a chance to test whether an approval overrides a condemnation.

**Pull request merged**